### PR TITLE
Detailed installation instructions

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Taken from https://github.com/Codeception/Codeception/pull/4233

### CODE REVIEW

File: docs/modules/WebDriver.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ To run Selenium Server you will need Java and Chrome or Firefox browser installe
 
 ### PhantomJS
 
-PhantomJS is a headless alternative to Selenium Server that implements
+PhantomJS is a [headless browser](https://en.wikipedia.org/wiki/Headless_browser) alternative to Selenium Server that implements
```
Comment (DavertMik) [inline docs/modules/WebDriver.md]: Uuuuuups!
This documentation generated from `src/Codeception/Module/WebDriver.php`
Edit a docblock of a class to change the documentation. 
On each release docblocks are transformed into markdown and published on site

Comment (ThomasLandauer) [inline docs/modules/WebDriver.md]: OK, see https://github.com/ThomasLandauer/Codeception/commit/77fe6ef59df29a2293e7e589ffe62de8b66b94f0

* Is it really a good idea to have hundreds of lines of documentation inside the source code?
* I'm losing track of my commits ;-) Could you merge them and/or invoke the markdown-conversion manually?

File: ext/README.md [hunk 2]
```diff
@@ -27,17 +27,14 @@ extensions:
 
 ## Codeception\Extension\Recorder
 
-Saves screenshots of each step in acceptance tests and shows them as a slideshow.
+Saves a screenshot of each step in acceptance tests and shows them as a slideshow on one HTML page.
```
Comment (DavertMik) [inline ext/README.md]: Like for modules, this should go to `ext/Recorder.php` file. Inside dockblock

File: docs/modules/WebDriver.md (hunk 1 above)
Comment (DavertMik) [inline docs/modules/WebDriver.md]: > Is it really a good idea to have hundreds of lines of documentation inside the source code?

Yes. At least you always have one source of thruth :+1: 

> I'm losing track of my commits ;-) Could you merge them and/or invoke the markdown-conversion manually?

I will lose track too. Could you just copy your changes to docblocks of WebDriver and Recorder classes, and I will merge it

**Comment by DavertMik** [comment]
Ok, I need to merge it now as I'm preparing the release and I'd really like to see this new updates on a website. Thanks for making the documentation better! :1st_place_medal:

File: docs/modules/WebDriver.md (hunk 1 above)
Comment (DavertMik) [inline docs/modules/WebDriver.md]: Oh yes, I see, some changes indeed are in src/Codeception/Module/WebDriver  :)

**Pull request merged**